lib/id3c/cli/command/location.py

```diff
@@ -335,7 +334,7 @@ def lookup(scale,
     location identifier as <scale>_identifier. Original input can be dropped
     with the --drop-input-columns flag.
     """
-    input_df = load_lookup_input(filename)
+    input_df = load_input_from_file_or_stdin(filename, SUPPORTED_EXTENSIONS)
```

Comment:
I have the same concerns here as the other load function: 
1. not to change functions while moving them, and 
2. that this falsely gives the illusion of being able to support more extensions than it actually does. (reality is we only support csv or tsv within the body of the function)